README.md

```diff
@@ -27,6 +27,7 @@ Make intuitve pixel art in this simple drawing tool written in _vanilla javascri
 | undo             | Z                   |
 | redo             | X                   |
 | color swap       | C                   |
+| delete selected area | Del                 | 
```

Comment:
Can you move this line just above `remove selection`?

As this list is getting long, it would be best to group similar operations together.

Reply:
I changed the line and added Backspace to the hotkeys column